### PR TITLE
DB stuff

### PR DESCRIPTION
**Pull request by mabbus**
- migrations refactor
- db restructure

### CODE REVIEW

File: database/factories/BusinessesFactory.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        'legal_name' => $faker->company,
+        'dba_name' => $faker->company,
+        'business_email' => $faker->companyEmail,
+        'business_address' => $faker->streetAddress,
+        'business_city' => $faker->city,
+        'business_state' => ucwords(substr($faker->state(), 0, 2)),
+        'business_zip' => $faker->postcode,
+        'fein' => $faker->randomNumber(9, true),
+        'business_phone' => $faker->e164PhoneNumber,
+        'entity_type' => $faker->randomElement(['LLC', 'SCORP', 'non-profit']),
```
Comment (lexod) [inline database/factories/BusinessesFactory.php:17]: these should be defined in model as constants

File: database/factories/DealFactory.php [hunk 2]
```diff
@@ -10,15 +10,15 @@
...
-
+    $workflowStatus = \App\WorkflowStatus::all();
     $terms = $faker->randomNumber(1);
 
     return [
-        'merchant_id' => $merchants->count() ? $merchants->random()->id : 0,
-        'company_id' => $companies->count() ? $companies->random()->id : 1,
-        'status' => $faker->randomElement(config('entity.deal.statuses')),
+        'business_id' => $business->count() ? $business->random()->id : 0,
+        'company_id' => 1,
```
Comment (lexod) [inline database/factories/DealFactory.php:20]: take random Company

File: database/factories/WorkflowsFactory.php [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+use Faker\Generator as Faker;
+
+
+$factory->define(\App\Workflow::class, function (Faker $faker) {
+    return [
+        'name' => $faker->text(10),
+        'idx' => $faker->randomNumber(1),
+        'company_id' => 1,
```
Comment (lexod) [inline database/factories/WorkflowsFactory.php:10]: take random company

File: database/factories/WorkflowsStatusFactory.php [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+use Faker\Generator as Faker;
+
+
+$factory->define(\App\WorkflowStatus::class, function (Faker $faker) {
+    return [
+        'name' => $faker->text(10),
+        'idx' => $faker->randomNumber(1),
+        'workflow_id' => 1,
```
Comment (lexod) [inline database/factories/WorkflowsStatusFactory.php:10]: take random workflow

File: database/migrations/2020_08_05_195656_create_merchants_table.php [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+            $table->foreign('user_id')->references('id')->on('users');
+            $table->string('first_name', 20);
+            $table->string('middle_name', 20)->nullable()->default(null);
+            $table->string('last_name', 20);
+            $table->string('job_title', 20);
+            $table->string('email', 255);
+            $table->string('address', 100);
+            $table->string('city', 100);
+            $table->string('state', 2);
+            $table->string('zip', 10);
```
Comment (lexod) [inline database/migrations/2020_08_05_195656_create_merchants_table.php:28]: zip should be 5 digits unless we are doing extended

File: database/migrations/2020_08_05_195656_create_merchants_table.php [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+            $table->string('ssn', 9);
+            $table->string('drivers_license', 30);
+            $table->string('ownership',10);
+            $table->string('phone', 15);
```
Comment (lexod) [inline database/migrations/2020_08_05_195656_create_merchants_table.php:32]: phone should be 10 symbols (xxxxxxxxxx) or 12 symbols (xxx-xxx-xxxx) or 14 symbols(x-xxx-xxx-xxxx)

File: database/migrations/2020_08_05_203709_create_businesses_table.php [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+            function (Blueprint $table) {
+                $table->increments('id');
+                $table->string('legal_name', 100);
+                $table->string('dba_name', 100);
+                $table->string('business_email', 255);
+                $table->string('business_address', 100);
+                $table->string('business_city', 100);
+                $table->string('business_state', 2);
+                $table->string('business_zip', 10);
+                $table->string('business_phone', 15);
```
Comment (lexod) [inline database/migrations/2020_08_05_203709_create_businesses_table.php:26]: business_email?

File: database/migrations/2020_08_05_214838_create_deals_syndicates_table.php [hunk 8]
```diff
@@ -29,6 +29,9 @@ public function up()
             $table->double('amount', 100, 2);
             $table->unsignedDecimal('management_fee', 6,3)->comment('Participation fee for syndicates');
 
+            $table->boolean('show')
```
Comment (lexod) [inline database/migrations/2020_08_05_214838_create_deals_syndicates_table.php:32]: show is too abstract.
show_details or display_details

File: database/migrations/2020_08_05_215721_update_payment_profile_table.php [hunk 9]
```diff
@@ -13,9 +13,7 @@ class UpdatePaymentProfileTable extends Migration
      */
     public function up()
     {
-        Schema::table('payment_accounts', function (Blueprint $table) {
```
Comment (lexod) [inline database/migrations/2020_08_05_215721_update_payment_profile_table.php:16]: delete this file

File: database/factories/BusinessesFactory.php (hunk 1 above)
Comment (mabbus) [reply database/factories/BusinessesFactory.php:17]: Dont have the full list yet

**Pull request merged**